pkg/helper/client.go

```diff
@@ -38,16 +38,14 @@ const vlanFilteringCommand = "vlan-filtering"
 const defaultGwRetrieveTimeout = 120 * time.Second
 const defaultGwProbeTimeout = 120 * time.Second
 const apiServerProbeTimeout = 120 * time.Second
+const defaultInterfacesFilter = "veth*"
 
 var (
 	interfacesFilterGlob glob.Glob
 )
 
-func init() {
-	interfacesFilter, isSet := os.LookupEnv("INTERFACES_FILTER")
-	if !isSet {
-		panic("INTERFACES_FILTER is mandatory")
-	}
+func InitClient(interfacesFilter string) {
```

Comment:
Let's be more specific here and call it SetInterfacesFilter? That's all it does.